UCLALibrary/prl-harvester
# [SERV-569] Add Localstack S3 container for testing

To test:

1. Spin up the containers with `mvn clean compile docker:run -Ddocker.noCache -Ddocker.showLogs`
2. Send an HTTP GET request to `localhost:$\{test.s3.port\}/thumbnails-source/ark+=21198=zz00253w02.jpg` and confirm a successful response
